Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -289,3 +274,31 @@ open class Player: UIViewController {
         NotificationCenter.default.removeObserver(self)
     }
 }
+
+extension Player {
+    private var isPaused: Bool {
+        return activePlayback?.state == .paused
+    }
+
+    private var isPlaying: Bool {
+        return activePlayback?.state == .playing
+    }
+
+    open override func pressesBegan(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
```

Comment:
An override func in a `extension class` feels strange, don't you think? What about making this extension be like `extension class: superClass`  or moving this func to the class implementation?

Reply:
The main reason for doing such a thing was to keep the context of the deleted class `DecoratedAVPlayerViewController`. This code is not meant to be used within the Player itself, so that's the main reason to keep it separate in an extension. 

What do you think?

Reply:
I usually avoid to create extensions without a protocol to conform, like `extension OwnMyClass`, I move funcs to the class itself. But we don't have a team agreement for this, so it's up 2 u. = )